src/main/resources/org/jenkinsci/plugins/schedulebuild/ScheduleBuildAction/index.jelly

```diff
@@ -5,7 +5,7 @@
         <l:main-panel>
             <h1>${%Project} ${it.owner.name}</h1>
             <f:form name="schedule" action="next" method="post">
-                <f:entry title="${%BuildOn}:" field="date">
+                <f:entry title="${%BuildOn} (mm/dd/yy):" field="date">
```

Comment:
Hi @ArcticJuggernaut,
just tested this in Version 1.0.0-beta1 of this plugin. The displayed date in the field will use the browsers locale/date format settings. So, for me it's dd.mm.yy... I think the best would be to have this text added dynamically based on the browser settings, but for now I would revert this change. OK for you?

Reply:
Yep sure thing.